CHANGELOG.develop

```diff
@@ -2030,7 +2030,10 @@ Other:
 - Use =python-mode= for SCons script files (thanks to shanemikel)
 - Added LSP support, which can be used by enabling the =lsp= layer and setting
   the =python= layer's =python-backend= variable to =lsp=
-  (thanks to Yuan Fu and Sylvain Benner)
+  (thanks to Yuan Fu and Sylvain Benner). The LSP backend can use either the
```

Comment:
Fill issue. Start on a newline below the previous thanks parentheses. With a leading `-` (dash).

Reply:
Had been trying to keep the python LSP stuff in a single changelog entry, but split now.